src/site/stages/build/index.js

```diff
@@ -38,6 +39,54 @@ const addSubheadingsIds = require('./plugins/add-id-to-subheadings');
 
 function defaultBuild(BUILD_OPTIONS) {
   const smith = Metalsmith(__dirname); // eslint-disable-line new-cap
+
+  // Override the normal use function to provide timing and logging information
+  smith._use = smith.use;
+  let stepCount = 0;
+  smith.use = function use(plugin, description) {
+    stepCount++;
+    if (!description) return smith._use(plugin);
+
+    // Wrapped in yet another function so we can get the step count
+    return (step =>
+      smith._use(async (files, metalsmith, done) => {
+        /* eslint-disable no-console */
+        console.log(chalk.cyan(`\nStep ${step} start: ${description}`));
+        const timer = process.hrtime();
+
+        let calledDone = false;
+        const logAndDone = err => {
+          calledDone = true;
+          const time = process.hrtime(timer)[1] / 1000000;
+
+          // Color the time
+          let color;
+          if (time < 1000) color = chalk.green;
+          else if (time < 10000) color = chalk.yellow;
+          else color = chalk.red;
+          const coloredTime = color(`[${time}ms]`);
+
+          console.log(
+            chalk.cyan(`Step ${step} end ${coloredTime}: ${description}`),
+          );
+          done(err);
+        };
+        // Plugins can call done() or not. If they don't, they'll
+        // return a promise, which we'll want to call our function on.
+        const res = plugin(files, metalsmith, logAndDone);
+        console.log(`After plugin call for step ${step}: ${description}`);
+        if (res instanceof Promise) {
```

Comment:
`async` Metalsmith functions such as [`getDrupalContent`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/drupal/metalsmith-drupal.js#L202) return a promise, but...probably doesn't actually warrant this being here. Probably?